YellowNotes.Api/YellowNotes.Api.csproj

```diff
@@ -4,9 +4,21 @@
     <ProjectReference Include="..\YellowNotes.Core\YellowNotes.Core.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Controllers\" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="3.1.1" />
```

Comment:
do we need this? this is for what? I'm asking because I've never seen it before.

Reply:
we dont need this :p